source/features/notes/RandomNotePage.module.css

```
@@ -1,159 +1,148 @@
-
-.article :global(.note-meta-date) {
-  opacity: 0.7;
-}
-
-.article :global(.note-details) {
-  display: none;
-}
+  display: grid;
+  row-gap: var(--space-8);
+
+  :global(.note-heading) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)